logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserRepository.kt

```diff
@@ -407,6 +409,10 @@ internal class UserDataSource internal constructor(
         userDAO.updateUserAvailabilityStatus(userId.toDao(), availabilityStatusMapper.fromModelAvailabilityStatusToDao(status))
     }
 
+    override suspend fun updateSupportedProtocols(protocols: Set<SupportedProtocol>): Either<CoreFailure, Unit> {
```

Comment:
Shouldn't this operation update the local value in the database as well ? 🤔

Reply:
it should! 👍

Reply:
Done